lighthouse-core/gather/drivers/driver.js

```diff
@@ -492,7 +492,8 @@ class Driver {
   beginEmulation() {
     return Promise.all([
       emulation.enableNexus5X(this),
-      emulation.enableNetworkThrottling(this)
+      emulation.enableNetworkThrottling(this),
+      //emulation.enableCPUThrottling(this),
```

Comment:
@addyosmani is this good enough? https://github.com/GoogleChrome/lighthouse/pull/747/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R135